tests/dummy/app/routes/query.ts

```diff
@@ -14,7 +18,7 @@ export default class QueryRoute extends Route {
   public async model(): Promise<ArrayProxy<GroupModel>> {
     return this.store.query('group', {
       filter(reference: firebase.firestore.CollectionReference) {
```

Comment:
The correct type here would now be `CollectionReference` from the modular SDK rather than `firebase.firestore.CollectionReference`.